Docs/source/usage/parameters.rst

```diff
@@ -1558,6 +1558,8 @@ Numerics and algorithms
     Available options are: ``direct``, ``esirkepov``, and ``vay``. The default choice
     is ``esirkepov`` for FDTD maxwell solvers and ``direct`` for standard or
     Galilean PSATD solver (that is, with ``algo.maxwell_solver = psatd``).
+    Note that ``vay`` is only available for ``algo.maxwell_solver = psatd`` and not available for
+    ``algo.maxwell_solver = yee`` or ``algo.maxwell_solver = ckc``.
```

Comment:
Here I would maybe only state "Note that `vay` is only available for `algo.maxwell_solver = psatd`", without stating the options for which it is not available, otherwise we risk missing some of those as new ones are added.